inginious/frontend/installer.py

```diff
@@ -512,12 +520,33 @@ def configure_authentication(self, database):
 
             self._display_info("You can choose an authentication plugin between:")
             self._display_info("- 1. LDAP auth plugin. This plugin allows to connect to a distant LDAP host.")
+            self._display_info("- 2. SAML auth plugin. This plugin allows to connect to a distant SAML Identity Provider.")
 
             plugin = self._ask_with_default("Enter the corresponding number to your choice", '1')
-            if plugin not in ['1']:
-                continue
-            elif plugin == '1':
+            if plugin == '1':
                 options["plugins"].append(self.ldap_plugin())
+            elif plugin == '2':
+                self._display_warning("Please consult the documentation to add this plugin.")
+            else:
+                continue
+
+        options["session_parameters"] = {
+            "timeout": 86400,  # 24 * 60 * 60, # 24 hours in seconds
+            "ignore_change_ip": False,
+            "secret_key": "fLjUfxqXtfNoIldA0A0G",
+            "secure": False
+        }
```

Comment:
Is this necessary as the parameters are asked below with default values ? This is a bit confusing.